src/pages/profiles/ProfileSelector.tsx

```diff
@@ -70,109 +36,66 @@ const ProfileSelector: FC<Props> = ({ project, selected, setSelected }) => {
     (profile) => !selected.includes(profile.name)
   );
 
+  const addProfile = () => {
+    const nextProfile = unselected.pop()?.name;
+    if (nextProfile) {
+      setSelected([...selected, nextProfile]);
+    }
+  };
+
   return (
     <>
-      <Row>
-        <Col size={12}>
-          <Label className="u-sv-1">Apply profiles</Label>
-          <div className="u-text--muted u-sv1">
-            Add profiles and create a hierarchy
-          </div>
-        </Col>
-      </Row>
-      <Row className="p-profile-select">
-        <Col size={6} className="p-profile-select-box">
-          <i className="p-profile-select-title">Available profiles</i>
-          <hr />
-          <div className="p-profile-select-list">
-            <List
-              items={unselected.map((profile) => (
-                <Fragment key={profile.name}>
-                  <span
-                    className="p-profile-select-name u-truncate"
-                    title={profile.name}
-                  >
-                    {profile.name}
-                  </span>
-                  <div className="u-float-right">
-                    <Button
-                      dense
-                      hasIcon
-                      small
-                      type="button"
-                      onClick={() => toggleProfile(profile)}
-                    >
-                      Apply
-                    </Button>
-                  </div>
-                </Fragment>
-              ))}
-            />
-          </div>
-        </Col>
-        <Col size={6} className="p-profile-select-box">
-          <i className="p-profile-select-title">Applied profiles</i>
-          <hr />
-          <div className="p-profile-select-list">
-            <List
-              items={selected.map((name, i) => {
-                const profile = profiles.find(
-                  (profile) => profile.name === name
-                );
-                if (!profile) {
-                  return null;
+      {selected.map((value, i) => (
+        <Row key={value}>
+          <Col size={8}>
+            <Select
+              help={
+                i > 0 &&
+                "Additional profile, overrides settings specified through profiles above"
```

Comment:
Do we really need to show this help text below every added profile? Wouldn't it be enough to show it only on the last one?

Reply:
You are right, having it only on the last element is a good idea.